app/domain/authentication/security/validate_webservice_is_authenticator.rb

```diff
@@ -21,7 +21,7 @@ def validate_webservice_is_configured_authenticator
       end
 
       def webservice_id
-        "#{@webservice.authenticator_name}/#{@webservice.service_id}"
+        @webservice.service_id.blank? ? @webservice.authenticator_name : "#{@webservice.authenticator_name}/#{@webservice.service_id}"
```

Comment:
Maybe in this place you should call function that checks if the authenticator is with mandatory for service id?